runtime.md

```diff
@@ -1,10 +1,5 @@
 # Runtime and Lifecycle
 
-## Scope of a Container
-
-Barring access control concerns, the entity using a runtime to create a container MUST be able to use the operations defined in this specification against that same container.
-Whether other entities using the same, or other, instance of the runtime can see that container is out of scope of this specification.
-
```

Comment:
On Mon, Feb 22, 2016 at 06:40:10AM -0800, Doug Davis wrote:
> > -## Scope of a Container
> > -
> > -Barring access control concerns, the entity using a runtime to create a container MUST be able to use the operations defined in this specification against that same container.
> > -Whether other entities using the same, or other, instance of the runtime can see that container is out of scope of this specification.
> > -
>
> not sure why you removed this.  I thought you were asking for text
> related to the scope of a runtime's data.

Yup, and I added that text here [1], and bound it to all actions here
[2].

[1]: https://github.com/wking/opencontainer-specs/commit/2fff5454fb7f20beec6110ff8725fc8507f4cce8#diff-b84a8d65d8ed53f4794cd2db7e8ea731R68
[2]: https://github.com/wking/opencontainer-specs/commit/2fff5454fb7f20beec6110ff8725fc8507f4cce8#diff-b84a8d65d8ed53f4794cd2db7e8ea731R53